packages/headless/src/features/facets/facet-set/facet-set-analytics-actions-utils.test.ts

```diff
@@ -106,7 +106,7 @@ describe('facet-set-analytics-action-utils', () => {
           field,
           currentValues: [
             buildMockCategoryFacetValueRequest({
-              state: 'selected',
+              state: 'idle',
```

Comment:
I decided to changes this, because I don't think it make sense:
if the child value is selected, the parent value should not.

At least, that's what I saw in the response & requests when playing with atomic in stackblitz